modules/nf-core/quilt/quilt/main.nf

```diff
@@ -0,0 +1,62 @@
+process QUILT_QUILT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::r-quilt=1.0.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-quilt:1.0.4--r43h06b5641_2':
+        'biocontainers/r-quilt:1.0.4--r43h06b5641_2' }"
+
+    input:
```

Comment:
It could be nice to have all the parameter with the same case convention.

Reply:
yes! I will correct that